### ISSUE TITLE
Question on graylog deployment

### ISSUE THREAD

**Issue by jtryan** [sherzberg/graylog-plugin-s3#21]
Hi,

I am looking at this as we want to pull S3 json logs into Graylog. I have heard this plugin only works with graylog instances deployed on AWS. Is this so? I do not want to start to troubleshoot something that cannot work. :) Thanks.

**Comment by nicholas-a-clark** [comment]
This is not true we have an in house Graylog setup and this works fine for us.
